velox/common/memory/MappedMemory.cpp

```diff
@@ -59,17 +60,57 @@ void MappedMemory::Allocation::findRun(
   VELOX_CHECK(false, "Seeking to an out of range offset in Allocation");
 }
 
-// Rounds 'value' to the next multiple of 'factor'.
-template <typename T, typename U>
-static inline T roundUp(T value, U factor) {
-  return (value + (factor - 1)) / factor * factor;
+MachinePageCount MappedMemory::ContiguousAllocation::numPages() const {
+  return bits::roundUp(size_, kPageSize) / kPageSize;
 }
 
 // static
 void MappedMemory::destroyTestOnly() {
   instance_ = nullptr;
 }
 
+std::string MappedMemory::toString() const {
+  return fmt::format("MappedMemory: Allocated pages {}", numAllocated());
+}
+
+MachinePageCount MappedMemory::allocationSize(
```

Comment:
This function seems to always allocate pages of the same size and set numSizes to 1. Is this intentional? If so, let's simplify the interface to return number of pages and size class.

Reply:
@oerling I wonder if this question / comment got lost somehow. Same for other questions and comments below.

------
This function seems to always allocate pages of the same size and set numSizes to 1. Is this intentional? If so, let's simplify the interface to return number of pages and size class.